### PR TITLE
docs: misc fixes

### PR DESCRIPTION
**Pull request by kevinthecheung**
Description here... 

Checklist (if applicable):
- [ ] Tested (manually, unit tested, etc.)
- [ ] Changelog updated
- [ ] Docs updated

### CODE REVIEW

File: docs/migrating-from-0.5.md [hunk 1]
```diff
@@ -14,53 +14,36 @@ npm i -g genkit-cli
...
 ```
 
-This command starts both the flow server and the Dev UI in a single command.
-
-**New Commands:**
-
-Now, you must start the dev UI and the flow servers as separate steps.
-
-To start the dev UI:
+Starts your Genkit application code ONLY:
```
Comment (pavelgj) [inline docs/migrating-from-0.5.md:33]:
```suggestion
Starts ONLY your application code in Genkit dev mode:
```

File: docs/migrating-from-0.5.md [hunk 2]
```diff
@@ -14,53 +14,36 @@ npm i -g genkit-cli
...
+```
+Starts the Dev UI ONLY:
 
 ```posix-terminal
-GENKIT_ENV=dev tsx --watch path/to/index.ts
+genkit start
 ```
 
-Starting the flow server this way makes it easier for you to attach a debugger to your code. Be sure to set `GENKIT_ENV=dev` in your debugger’s startup configuration.
+Previously, the `genkit start` command would start the Dev UI and Flow Server together. If you have any CI/CD pipelines relying on this command, you may need to update the pipeline.
```
Comment (pavelgj) [inline docs/migrating-from-0.5.md:46]:
```suggestion
Previously, the `genkit start` command would start the Dev UI and your application code together. If you have any CI/CD pipelines relying on this command, you may need to update the pipeline.
```

File: docs/tool-calling.md [hunk 3]
```diff
@@ -214,7 +214,7 @@ const getWeather = ai.defineTool(
 );
 
 const generateOptions: GenerateOptions = {
-  prompt: 'What's the weather like in Baltimore?',
+  prompt: "What's the weather like in Baltimore?",
```
Comment (pavelgj) [inline docs/tool-calling.md:217]: consistency -- single quote are used above.

**Review by pavelgj — APPROVED** [review]

File: docs/tool-calling.md (hunk 3 above)
Comment (kevinthecheung) [reply docs/tool-calling.md:217]: but the apostrophe

Comment (pavelgj) [reply docs/tool-calling.md:217]: ah, all good then!

**Pull request merged**